cpp/CMakeLists.txt

```diff
@@ -149,20 +152,27 @@ if (enable-test)
   include_directories(${GTEST_INCLUDE_DIRS})
 
   # Add source files according to machine setup
-  if (CUDA_HAVE_GPU)
+  if (CUDA_AND_GPU)
+  message("MAKING GPU TESTS")
     file(GLOB_RECURSE TEST_SRC_FILES RELATIVE
          ${PROJECT_SOURCE_DIR}
          test/cpu_operations_test/*.cc
-         test/util_test/*.cc
+         test/logistic_regression_test/*.cc
```

Comment:
Could you move all logistic regression related tests to NICE/cpp/test/model_test?

Reply:
Fixed.